### ISSUE TITLE
Babel error during build

### ISSUE THREAD

**Issue by bvalyou** [mike-engel/floating-label-react#2]
When imported into my application, my webpack build fails on the babel config contained in this package.

> ERROR in ./node_modules/floating-label-react/index.js
> Module build failed: ReferenceError: Unknown plugin "add-module-exports" specified in "./node_modules/floating-label-react/package.json" at 0, attempted to resolve relative to "./node_modules/floating-label-react"

**Comment by mike-engel** [comment]
@bvalyou Can you paste your webpack config? Are you using babel in your project?

**Comment by bvalyou** [comment]
I am using babel.

```
const CleanWebpackPlugin = require('clean-webpack-plugin');
const ExtractTextPlugin = require('extract-text-webpack-plugin');
const HtmlWebpackPlugin = require('html-webpack-plugin');
const minimist = require('minimist');
const path = require('path');
const webpack = require('webpack');

//eslint-disable-next-line no-magic-numbers
const { watch } = minimist(process.argv.slice(2));

const config = {
	devtool: watch ? 'source-map' : false,
	entry: {
		initial: './client',
		vendor: ['react', 'react-dom']
	},
	output: {
		path: path.resolve(__dirname, 'public/dist/'),
		filename: '[name].bundle.[chunkhash].js',
		chunkFilename: '[name].bundle.[chunkhash].js',
		publicPath: '/public/dist/'
	},
	module: {
		rules: [
			{ test: /\.js$/, use: 'babel-loader' },
			{
				test: /\.css$/,
				use: ExtractTextPlugin.extract({
					fallback: 'style-loader',
					use: {
						loader: 'css-loader',
						options: { modules: true, importLoader: 1, sourceMap: watch }
					}
				})
			}
		]
	},
	plugins: [
		new CleanWebpackPlugin('public/dist'),
		new webpack.optimize.CommonsChunkPlugin({
			names: ['main'],
			children: true
		}),
		new webpack.optimize.CommonsChunkPlugin({
			names: ['vendor', 'manifest']
		}),
		new HtmlWebpackPlugin({
			template: 'webpack/template.ejs',
			filename: '../../views/index.hbs'
		}),
		new ExtractTextPlugin('styles.css')
	]
};

module.exports = config;
```

**Comment by bvalyou** [comment]
My .babelrc might be helpful as well:

```
{
	"presets": [
		[ "env", {
			"targets": {
				"browsers": [
					"last 2 versions"
				]
			},
			"modules": false
		} ],
		"react"
	],
	"plugins": [
		"transform-object-rest-spread",
		"syntax-dynamic-import"
	]
}
```

**Comment by mike-engel** [comment]
@bvalyou Have you tried excluding `node_modules` from your webpack babel loader?

```
rules: [
		{
			test: /\.js$/,
			use: 'babel-loader'
			exclude: /(node_modules)/
		},
		{
			test: /\.css$/,
			use: ExtractTextPlugin.extract({
				fallback: 'style-loader',
				use: {
					loader: 'css-loader',
					options: { modules: true, importLoader: 1, sourceMap: watch }
				}
			})
		}
	]
```

**Comment by bvalyou** [comment]
@mike-engel That is certainly something I neglected in my webpack config, cheers! Unfortunately, now I get the below error. I would recommend updating the build to export, at the very least, (currently) valid ESModule files with an additional directory containing raw source. That syntax is something I might add to my babel config soon, but it probably shouldn't be a requirement of the lib.


> ERROR in ./node_modules/floating-label-react/index.js
> Module parse failed: Unexpected token (46:8)
> You may need an appropriate loader to handle this file type.
> | 
> | class FloatingLabel extends React.Component {
> |   state = {
> |     floating: false,
> |     focused: false,

**Comment by mike-engel** [comment]
Hi @bvalyou. Unfortunately I don't have much time to dedicate to this project. If you'd like to throw together a PR, however, I would be happy to review and merge it. Thanks!

**Comment by bvalyou** [comment]
The root issue is that the npm package isn't actually exporting the output of your build script, it's spitting out the raw source.

**Comment by bvalyou** [comment]
I think you forgot to build before publishing 0.1.1, 0.1.0 looks fine

**Comment by mike-engel** [comment]
@bvalyou Thanks for looking into that. I've published 0.1.2 which should fix the problem. Thanks!
